### DISCUSSION TITLE
setting date and time with dosemu2 and fdpp

### DISCUSSION THREAD

**Discussion by nromey** [dosemu2/comcom64#80] [General]
Hi guys,

I'm working with a dos application which requires that I change the date to pre y2k to set it up and then do maintenance.  Then I can set the time back to normal.  I've set 
$_timemode = "bios"
in a custom dosemu.conf which I load at start.
When I type date 10/01/1999
it just repeats my current date.
Do I need to try to temporarily run dosemu2 through freedos to allow this behavior? I do know that dosemu allowed me to set time and date.  What am I missing?

**Comment by stsp** [comment]
Its likely not supported by comcom32.
Try with freecom, and if it works there,
then open a comcom32 bug.
comcom32 is a bit of a "loose end" so far..

**Reply by nromey** [reply]
Is freecom just the command.com located in the comcom32 directory i share?

**Reply by stsp** [reply]
Symlink freecom.exe into ~/.dosemu/drive_c/command.com
That's all.

**Reply by nromey** [reply]
Took a bit to find freecom. Ran it, again with time setting to bios, with freecom, I was able to set time and date.  I'll file a bug if that'll help.

NER

**Comment by nromey** [comment]
Hi,

I was able to actually just set time and date manually in Linux and I think it worked that way. I do know that with dosemu/freedos, I was able to do it. What's the easiest way to install and/or use freedos "freecom" rather than comcomm32.  Is there a document on the procedure, and can I switch back to comcom32 after testing? Sorry for the questions, I'm new to this.
